atom/browser/native_window_mac.mm

```diff
@@ -934,6 +934,8 @@ - (void)drawRect:(NSRect)dirtyRect {
 }
 
 void NativeWindowMac::ShowDefinitionForSelection() {
```

Comment:
We can remove this implementation and add a helper function in `browser-window.js`.